usbsdmux/__main__.py

```diff
@@ -36,10 +36,19 @@ def main():
         "dut - set to dut mode\n"
         "client - set to dut mode (alias for dut)\n"
         "host - set to host mode\n"
-        "off - set to off mode",
-        choices=["get", "dut", "client", "host", "off"],
+        "off - set to off mode\n"
+        "gpio0 - manipulate gpio0\n"
+        "gpio1 - manipulate gpio1",
+        choices=["get", "dut", "client", "host", "off", "gpio0", "gpio1"],
         type=str.lower,
     )
+    parser.add_argument(
+        "gpio_action",
+        help="Action to perform on the GPIOs",
+        nargs="?",
+        default="",
+        choices=["", "low", "0", "high", "1", "get"],
+    )
```

Comment:
Shouldn't this be in a [subparser](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_subparsers)? Otherwise, the gpio_action argument would always be accepted, even for the existing "commands".